recipes/nifti_clib/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "nifti_clib" %}
+{% set version = "2.0.0" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/NIFTI-Imaging/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 62655d8457bd04ffd80cf2f07fe8c6aac2b8b00a69fbe71f399e3d9623128630
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - expat
+    - zlib
+
+test:
+    commands:
```

Comment:
You should add a test command. If there are only library/headers created, you can check if those files exist, e.g.:
```
test:
  commands:
    - test -f ${PREFIX}/include/directory/file.h
    - test -f ${PREFIX}/lib/directory/file.a
```
where `directory`, `file` are the paths to the copied headers/libraries as part of `nifti_clib`.